.github/workflows/regression-tests.yml

```diff
@@ -136,6 +140,20 @@ jobs:
           path: core/test-results/
           retention-days: 30
 
+      - name: Send slack notification
+        uses: slackapi/slack-github-action@v1.26.0
+        if: failure()
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+        with:
+          payload: |
+            {
```

Comment:
With the current setup we have for deployment, it is not possible to map the github user and bc user to tag them and so, we are not sending that information as part of the payload.

Reply:
Hey @avattipalli, it looks good! Is this possible to add the specific test run link in addition to the PR link? For example, https://github.com/bigcommerce/catalyst/actions/runs/10280244306

As if some check is failed on PR, we will go to this PR and look for this type of information to investigate the test run report, so would be really convenient to have this link directly in the Slack notification about test failure.

Reply:
That's a good callout, let me try and add that.